### DISCUSSION TITLE
Failed to run on Jetson AGX Orin

### DISCUSSION THREAD

**Discussion by zxaall** [ggml-org/llama.cpp#1677] [Q&A]
Code compiles fine. Fails to run with modes quantized on Linux Intel CPU. I re-run quantization on Jetson and got a file with different size. Now it sort of runs, but outputs only spaces and new lines. Conversion to f16 produces identical files on both systems.
Any ideas? I suspect some types have different sizes. And different types are used for what is supposed to be the same thing. Most likely somewhere in metadata. This results in different offsets in binary files. The same behavior with and without GPU support compiled.

**Comment by raj-khare** [comment]
@zxaall can you please tell how did you compile it?
